propertyestimator/__init__.py

```diff
@@ -24,6 +25,10 @@
         formatted_exception = traceback.format_exception(None, e, e.__traceback__)
         logging.warning(f'Could not load the {entry_point} plugin: {formatted_exception}')
 
+# Set up pint.
+from pint import UnitRegistry
+unit = UnitRegistry()
```

Comment:
This is very clever, but I worry that it'll be really confusing for a contributor some day.

Reply:
Unfortunately this is just the pint way of doing things: https://pint.readthedocs.io/en/0.9/tutorial.html#using-pint-in-your-projects